### PR TITLE
[FR] Update "group info" meta

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the meta page "group info".

New PR following an error with the old one: #5218

**Notify me if the PR is merge without review, that I self-review before.**

### CODE REVIEW

File: meta/group-info/fr.yaml [hunk 1]
```diff
@@ -1,13 +1,64 @@
-outdated: true # missing entries
 separator: ", "
+alumni:
+  roles:
+    academy: fondateur de osu!academy
```
Comment (geshii) [inline meta/group-info/fr.yaml:4]: you should put "fondateur de la osu!academy" I think

File: meta/group-info/fr.yaml [hunk 2]
```diff
@@ -1,13 +1,64 @@
...
+    beatmap: modérateur des beatmaps
+    chat: modérateur du tchat
+    forum: modérateur du forum
+    team: Équipe de présence officiel
+    dev: osu! development
+    player: support au joueur
     support: support technique
-    tournaments: organisation de tournois
+    tournaments: Gestionnaire des tournois
     wiki: administration du wiki
```
Comment (geshii) [inline meta/group-info/fr.yaml:22]: you should put capital letters on each new line, or not put them everywhere

Comment (DoryanR) [reply meta/group-info/fr.yaml]: C'est de base dans la version anglaise, je laisse comme tel, j'ai juste remit tout en minuscule

File: meta/group-info/fr.yaml [hunk 3]
```diff
@@ -1,13 +1,64 @@
...
+    tournaments: gestionnaire des tournois
     wiki: administration du wiki
+nat:
+  areas:
+    communication: communication
+    evaluation: évaluation
+    moderation: modération
+    structural: structurelle
+languages:
+  partial: un certaine <langue>
```
Comment (Satsukel) [inline meta/group-info/fr.yaml:30]:
```suggestion
  partial: une certaine <langue>
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

**Review by Satsukel — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**